functions/createNewUserAPI.js

```diff
@@ -32,10 +32,12 @@ exports.createNewUser = functions.https.onRequest((request, response) => {
                     photoURL: PhotoURL,
                     displayName: DisplayName,
                     email: Email,
-                    phoneNumber: PhoneNumber,
+                    phoneNumber: "",
```

Comment:
Better if we can check if there is a number inside PhoneNumber, is PhoneNumber is null then we must make it empty else we must update it with PhoneNumber